timetracker/vms/forms.py

```diff
@@ -130,6 +131,27 @@ def save(self, user):
             agency=agency
         )
 
+class CreateStaffEmployeeForm(forms.Form):
+    """
+    Form to create staffing employee.
+    """
+    agency_field=forms.ModelChoiceField(required=True,
```

Comment:
By default the name of the field is shown to the user, so if the name is something like `*_field`, add a `label=<human friendly name>` to the field parameters.